kld/src/database/payment.rs

```diff
@@ -150,6 +153,7 @@ impl Payment {
             fee: None,
             direction: PaymentDirection::Outbound,
             timestamp: SystemTime::now(),
+            bolt11: Some(invoice.bolt11.to_string()),
```

Comment:
The `lightning_invoice::Invoice` (bolt11 here) already includes a timestamp.  This will really confuse the user with a `SystemTime:now()`

Reply:
Do you know if we will let the user query his payment in the future?

If it is, what is the user expect with a timestamp query?

If not why do we store `amount` and `timestamp` here?
We already hold `Invoice`, which includes `amount` and `timestamp`.

Reply:
Okay, sorry about this, I am aware this timestamp is for a user, not the person who create the invoice.